### PR TITLE
Fat 2698 c350944

### PR DESCRIPTION
**Pull request by TetianaParanich**
***
https://issues.folio.org/browse/FAT-2698
***
https://foliotest.testrail.io/index.php?/cases/view/350944
***

### CODE REVIEW

File: cypress/support/fragments/data_import/mapping_profiles/newMappingProfile.js [hunk 1]
```diff
@@ -2,30 +2,32 @@ import { TextField, Button, Select, TextArea, Modal, HTML, including, MultiColum
...
   item: 'Item',
   invoice: 'Invoice',
   marcBib: 'MARC Bibliographic'
 };
-
 const organization = {
   gobiLibrary: 'GOBI Library Solutions',
   harrassowitz: 'Otto Harrassowitz GmbH & Co. KG',
 };
+const holdingType = {
```
Comment (MaksimTerekhov) [inline cypress/support/fragments/data_import/mapping_profiles/newMappingProfile.js:23]: 'holdingType' is assigned a value but never used.

File: cypress/support/fragments/data_import/job_profiles/newJobProfile.js [hunk 2]
```diff
@@ -140,6 +140,33 @@ export default {
...
+    ModalSelectActionProfile.selectActionProfile(matchProfileName, 'match');
+    cy.expect(Pane('New job profile').find(Accordion('Overview')).find(HTML(including(matchProfileName))).exists());
+    // link first action profile to match profile
+    cy.get('[id*="type-selector-dropdown-ROOT"]').eq(forMatchesOrder).click();
+    cy.do(actionsButton.click());
+    ModalSelectActionProfile.searchActionProfileByName(firstActionProfileName);
+    ModalSelectActionProfile.selectActionProfile(firstActionProfileName);
+    cy.expect(Pane('New job profile').find(Accordion('Overview')).find(HTML(including(firstActionProfileName))).exists());
+    // link second action profile to match profile
+    cy.get('[id*="type-selector-dropdown-ROOT"]').eq(forMatchesOrder).click();
```
Comment (IhorBohdan) [inline cypress/support/fragments/data_import/job_profiles/newJobProfile.js:157]: remoge selector to const

File: cypress/integration/data-import/match-on-pol-and-update-instance.spec.js [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+      permissions.uiInventoryViewCreateEditInstances.gui,
+      permissions.uiInventoryViewCreateEditItems,
+      permissions.uiInventoryViewInstances.gui,
+      permissions.uiQuickMarcQuickMarcBibliographicEditorView.gui
+    ])
+      .then(userProperties => {
+        user = userProperties;
+      })
+      .then(() => {
+        cy.login(user.username, user.password);
```
[the remark this answers is not in the document]
Comment (TetianaParanich) [reply cypress/integration/data-import/match-on-pol-and-update-instance.spec.js:115]: Done

File: cypress/integration/data-import/match-on-pol-and-update-instance.spec.js [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+        cy.getAdminToken();
+      });
+  });
+
+  after(() => {
+    cy.getInstance({ limit: 1, expandAll: true, query: `"title"=="${pol.title}"` })
+      .then((instance) => {
+        console.log(instance);
```
[the remark this answers is not in the document]
Comment (TetianaParanich) [reply cypress/integration/data-import/match-on-pol-and-update-instance.spec.js:123]: Done

File: cypress/support/fragments/data_import/mapping_profiles/newMappingProfile.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/mapping_profiles/newMappingProfile.js:23]: I deleted it

File: cypress/support/fragments/data_import/job_profiles/newJobProfile.js (hunk 2 above)
Comment (IhorBohdan) [reply cypress/support/fragments/data_import/job_profiles/newJobProfile.js:157]: no, only '[id*="type-selector-dropdown-ROOT"]' i mean

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/support/fragments/data_import/job_profiles/newJobProfile.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/job_profiles/newJobProfile.js:157]: I've created TODO for this changes

**Review by IhorBohdan — APPROVED** [review]

**Pull request merged**